src/crimson/os/seastore/random_block_manager/block_rb_manager.h

```diff
@@ -65,7 +65,7 @@ class BlockRBManager final : public RandomBlockManager {
     return device->get_journal_start() + device->get_journal_size();
   }
   size_t get_size() const final {
-    return device->get_available_size() - get_start_rbm_addr(); 
+    return device->get_available_size() - get_start_rbm_addr() - device->get_offset();
```

Comment:
what does device->get_offset() mean? I can't find it in main branch code.

Reply:
This means the beginning of each shard and is newly added by this PR.

Reply:
device->get_available_size() is super.size, right? so get_size() not get this shard size by this calculation, do you want get the shard size here?

Reply:
Yes, I missed something here. device->get_available_size() needs to return shard_info.size. So, this line should be `device->get_end() - get_start_rbm_addr()`. I will fix this.